plugins/git/plugin.zsh

```diff
@@ -29,6 +32,7 @@ PROMPT_GEOMETRY_GIT_CONFLICTS=${PROMPT_GEOMETRY_GIT_CONFLICTS:-false}
 PROMPT_GEOMETRY_GIT_TIME=${PROMPT_GEOMETRY_GIT_TIME:-true}
 PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT=${PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT:-false}
 PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY=${PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY:-true}
+PROMPT_GEOMETRY_GIT_STASHES=${PROMPT_GEOMETRY_GIT_STASHES:-false}
```

Comment:
can we name this `PROMPT_GEOMETRY_GIT_SHOW_STASHES` and set the default to `true` ?

Reply:
Sure!